src/buffer.c

```diff
@@ -501,51 +508,50 @@ buffer_set_initx(struct buffer_set_args *bsa)
         pthread_mutex_init(&bs->bs_mutex, NULL);
     }
 
-    int error = 0;
-
+    int rc = 0;
+    size_t off = 0;
+    size_t shifted = 0;
```

Comment:
Lines 530 and 531 declare tmp vars in the loop, can we please do the same w/ `shifted` since it's not used outside of the loop?